docs/source/api_ref_training.rst

```diff
@@ -51,3 +51,12 @@ of your finetuning job.
     :nosignatures:
 
     setup_torch_profiler
+
+Miscellaneous
+-------------
+
+.. autosummary::
+    :toctree: generated/
+    :nosignatures:
+
+    get_unmasked_sequence_lengths
```

Comment:
nit: I actually think this should belong in rlhf/sequence_processing.py and for classification relevant utils to be refactored once we support classification tasks. not blocking on this, up to you.